recorded_future/plugin.spec.yaml

```
@@ -86,7 +86,7 @@ types:
     timestamp:
       type: string
       required: false
-  risk:
+  risk_obj:
```

Same here, any reason we changed the name vs. keeping it the same?

thing_obj is not a standard convention we use.

```suggestion
  risk:
```

---

Need to update the remaining references

---

See above, this causes bugs sometimes.